OptimoveSDK/app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 31
     buildToolsVersion '30.0.3'
     defaultConfig {
-        applicationId "com.optimove.sdk.optimovemobilesdk"
+        applicationId "com.optimove.android.optimovemobilesdk"
```

Comment:
Any implication for google-services.json for test app?

Reply:
is there a test build app? where does it take googleservices from?

Reply:
Possibly a Q for @k-antipochkin

Reply:
Yes we have a test build app which is different then this one. It has an applicationId of its own so no problem here I guess